packages/cra-template-mephisto-review/template/src/Index.css

```diff
@@ -67,7 +67,7 @@ code {
   align-items: center;
   text-align: center;
   font-size: calc(10px + 1vmin);
-  margin-top: 50px;
+  padding-top: 50px;
```

Comment:
Helps prevent a nasty margin-collapsing bug I was running into when integrating `annotation-toolkit` into the review template. Basically the page always had a weird whitespace at the bottom and a forced scrollbar because of it, instead of nicely taking up 100% of the screen.